test/unit/eval_algorithms/test_general_semantic_robustness.py

```diff
@@ -1,457 +1,457 @@
-import pytest
```

Comment:
Why are these tests being commented out?

Reply:
Added the tests back, it was commented initially due to some failure.